spec/features/sign_up_for_Chitter_spec.rb

```diff
@@ -0,0 +1,12 @@
+feature 'Add user to Chitter' do
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)